### DISCUSSION TITLE
how to set up multiple worlds in tshock 1.4?

### DISCUSSION THREAD

**Discussion by pvini07BR** [Pryaxis/TShock#1798] [Q&A]
i have seen in the old forums for something like that but it says that its impossible to do, and had to create multiple servers, the old posts is so confusing and old, also i have seen popular servers (like pedguin's) that when you go to a world to another, you are teleported to space for a while and then teleport to the world, or there are other servers that you transform into a stone for a while and then teleports, i want to know how it works and how to do it. I dont know if its really multiple worlds in a single server, or i have to create multiple servers with different worlds, i have seen a plugin called "Dimensions" but it was confusing to set up, so any help?

**Comment by hakusaro** [comment]
This is still not possible. Dimensions is the best way to go.
